cs/TODO.md

```diff
@@ -0,0 +1 @@
+This should be moved to separate package, like `yii\yii2-cs`.
```

Comment:
> Should I create a repository for it and add you there with write permissions?

Yes, but not right now. I think we should accept this first - I don't want burn more time for this since there is an alternative: #13732

Also I suggest new repository - `yii2-coding-standards ` is for `PHP_Codesniffer`, we should not mix these 2 tools together.